[BD-3494] Update Live Activities FAQ (#8113)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md

### If Live Activities leverage push message functionality, do push notifications need to be enabled to receive Live Activities?

While Live Activities rely on push notifications for updates, they are controlled by different user settings. A user can opt into Live Activities but out of push notifications, and the other way around.

Live Activity update tokens expire after eight hours.

### Do Live Activities require push primers?

> [Push primers]({{site.baseurl}}/user_guide/message_building_by_channel/push/best_practices/push_primer_messages/) are a best practice to prompt your users to opt in to push notifications from your app. However, there is no system prompt to opt into Live Activities. Users are, by default, opted into Live Activities for any individual app when they install that app on iOS 16.1+. This permission can be disabled or re-enabled in the device settings on a per-app basis.

Reviewer: Ah I didn't realize this - so this is basically the "provisional" behavior comes by default for Live Activities?

Author: Yea basically if I install an app that has implemented Live Activities, the feature will be turned on by default. If I don't want to receive any Live Activities, I'll need to go into that app's settings on my phone and disable it.

> [Push primers]({{site.baseurl}}/user_guide/message_building_by_channel/push/best_practices/push_primer_messages/) are a best practice to prompt your users to opt in to push notifications from your app. However, there is no system prompt to opt into Live Activities. Users are, by default, opted into Live Activities for any individual app when they install that app on iOS 16.1+. This permission can be disabled or re-enabled in the device settings on a per-app basis.

Reviewer 2: ```suggestion
[Push primers]({{site.baseurl}}/user_guide/message_building_by_channel/push/best_practices/push_primer_messages/) are a best practice to prompt your users to opt in to push notifications from your app. However, there is no system prompt to opt into Live Activities. By default, users are opted into Live Activities for an individual app when the user installs that app on iOS 16.1 or later. This permission can be disabled or re-enabled in the device settings on a per-app basis.
```

## Technical topics and troubleshooting

### How do I know if Live Activities has errors?

Any Live Activity errors will be logged in the Braze dashboard in the [Message Activity Log]({{site.baseurl}}/user_guide/administrative/app_settings/message_activity_log_tab/), where you can filter by "LiveActivity Errors".

### After sending a push-to-start notification, why haven't I received my Live Activity?

> First, verify that your payload includes all the required fields described in the [`messages/live_activity/start`]({{site.baseurl}}/api/endpoints/messaging/live_activity/start) endpoint and that the `activity_attributes` and `content_state` fields match the properties defined in your project code. If you are certain that the payload is correct, you may be rate limited by APNs. This limit is imposed by Apple and not by Braze.

Reviewer 2: ```suggestion
First, verify that your payload includes all the required fields described in the [`messages/live_activity/start`]({{site.baseurl}}/api/endpoints/messaging/live_activity/start) endpoint. The `activity_attributes` and `content_state` fields should match the properties defined in your project's code. If you're certain that the payload is correct, its possible you may be rate-limited by APNs. This limit is imposed by Apple and not by Braze.
```

> To verify that your push-to-start notification successfully arrived at the device but was not displayed due to rate limits, you can debug your project using the Console app on your Mac. Attach the recording process for your desired device, and in the search bar, filter logs by `process:liveactivitiesd`.

Reviewer 2: ```suggestion
To verify that your push-to-start notification successfully arrived at the device but was not displayed due to rate limits, you can debug your project using the Console app on your Mac. Attach the recording process for your desired device, then filter the logs by `process:liveactivitiesd` in the search bar.
```
